audience_manager/form-tracker.rst

```diff
@@ -0,0 +1,101 @@
+.. highlight:: js
+.. default-domain:: js
+
+Audience Manager Form Tracker API
+=================================
+Form Tracker allows to gather data from all forms on your site in one place.
```

Comment:
It's unclear and confusing. It should say directly that it captures data from forms and saves it as a profile attribute of the visitor who submitted the form. The above sentence says that form data is saved and I can interpret it in very different ways.
Documentation should also explain how attributes are created (from names of fields), inform where attributes are saved and in what cases it can overwrite other existing attributes (i.e. those created based on custom variable or dimension). 
I think it would be also a good idea to link help center article that explains form tracking behavior.